lib/manager/poetry/artifacts.ts

```diff
@@ -43,28 +43,24 @@ export async function updateArtifacts(
     logger.debug(`Updating ${lockFileName}`);
     const cwd = join(config.localDir, subDirectory);
     const env = getChildProcessEnv();
-    let cmd: string;
+    let cmd = 'poetry';
+    const execOptions: ExecOptions = { cwd, env };
```

Comment:
Seems like `env` logic could be moved into `lib/util/exec` if it's always sourced from `getChildProcessEnv()` ?

Reply:
Not all exec call use `getChildProcessEnv` but should i think. So we should add this call to the `util/exec` logic.